docs/99-references/00-references.adoc

```diff
@@ -6,26 +6,20 @@
 == {references}
 
 // tag::DE[]
-Dieser Abschnitt enthält Quellenangaben, die ganz oder teilweise im Curriculum referenziert werden.
+Dieser Abschnitt enthält Ressourcen, die zur Erfüllung des Lehrplans verwendet werden sollten.
 // end::DE[]
 
 // tag::EN[]
-This section contains references that are cited in the curriculum.
+This section contains resources that should be used to fulfill the curriculum.
 // end::EN[]
 
-[NOTE]
-====
-Aufbau eines Eintrags-Ankers:
-```
-- [[[label,Text der erscheint]]]
-```
-ACHTUNG: Die Labels dürfen nur Buchstaben beinhalten, keine Zahlen oder Sonderzeichen
-
-= = =
-
-Structure of an anchor:
-```
-- [[[label,text that will be shown]]]
-```
-ATTENTION: labels have to be non-numeric.
-====
+- Brendan Burns: Designing Distributed Systems. Patterns and Paradigms for Scaleable, Reliable Services. O'Reilly Media, 2018
```

Comment:
Hier bitte für alle Quellen das Format verwenden, wie es in der gelöschten Note stand: 

`- [[[burns,Burns 2018]]] Brendan Burns: Designing Distributed Systems. Patterns and Paradigms for Scaleable, Reliable Services. O'Reilly Media, 2018`

Darauf verweisen könnt ihr dann in den jeweiligen Kapiteln via `<<burns>>`, im Text wird dann `Burns 2018` gerendert.

Reply:
Beim rendern sehen die Referenzen plötzlich so aus, hast du eine Idee was wir falsch machen?
```
- [[[davis,Davis 2019]]] Cornelia Davis: Cloud Native Patterns. Designing Change-tolerant Software. Manning, 2019

- [[[awati_wigmore,Awati, R. & Wigmore, I. (2022). Monolithic architecture]]] Awati, R. & Wigmore, I. (2022). Monolithic architecture. WhatIs.com. https://www.techtarget.com/whatis/definition/monolithic-architecture

```

![image](https://github.com/isaqb-org/curriculum-cloudinfra/assets/7222193/1ccf7537-4fbf-44e9-aba7-5fe944831d40)

Reply:
Die Referenzen sehen jetzt so aus:
![image](https://github.com/isaqb-org/curriculum-cloudinfra/assets/7222193/f8c1766c-a0c7-4b38-8764-4a795abc1667)